rules/windows/process_creation/win_malware_formbook.yml

```diff
@@ -22,10 +22,10 @@ detection:
         ParentCommandLine: 
             - 'C:\Windows\System32\\*.exe'
             - 'C:\Windows\SysWOW64\\*.exe'
-        CommandLine: 
-            - '* /c del "C:\Users\\*\AppData\Local\Temp\\*.exe'
-            - '* /c del "C:\Users\\*\Desktop\\*.exe'
-            - '* /C type nul > "C:\Users\\*\Desktop\\*.exe'
+        CommandLine|endswith: 
+            - ' /c del "C:\Users\\*\AppData\Local\Temp\\*.exe'
+            - ' /c del "C:\Users\\*\Desktop\\*.exe'
+            - ' /C type nul > "C:\Users\\*\Desktop\\*.exe'
```

Comment:
we need to remove asterisks ass we are using value modifiers here

Reply:
Fixed